src/SilverStripe/BehatExtension/Context/BasicContext.php

```diff
@@ -793,7 +793,7 @@ protected function getTable($selector) {
 			$selector)]/ancestor-or-self::table[1]");
 
         // Find tables by a .title node
-		$candidates += $page->findAll('xpath', "//table//*[@class='title' and contains(normalize-space(string(.)), 
+		$candidates += $page->findAll('xpath', "//table//*[contains(@class, 'title') and contains(normalize-space(string(.)), 
```

Comment:
How sensitive should this be? This will also match something with a class of "not-the-title". Correct xpath for class matching is https://blog.pivotal.io/labs/labs/xpath-css-class-matching